src/pspm_time2index.m

```diff
@@ -4,14 +4,14 @@
 %   to a sample index.
 % ● Format (optional arguments in []; all arguments up the last specified
 %           one need to be specified)
-%   index = pspm_time2index(time, sr [, data_length, zero_permitted, events])
+%   index = pspm_time2index(time, sr [, data_length, is_duration, events])
 % ● Arguments
 %           time: [vector or matrix] time stamps in second.
 %             sr: [numeric] sampling rate or frequency
 %    data_length: [integer] the length of data, which the index should
 %                 should not exceed
-%  zero_permitted: [0/1] whether zero is permitted (set to 0 for index and
-%                  to 1 for durations; default 0)
+%    is_duration: [0/1] whether an index or a duration is required, default
```

Comment:
I am not sure I understand what `is_duration` here means... it seems to be something similar to `zero_permitted` but with a different name?

Reply:
yes, renamed it for clarity.